lib/Twig/Extension/Templating/HeadLink.php

```diff
@@ -123,7 +123,7 @@ public function __construct(
      *
      * @return HeadLink
      */
-    public function __invoke(array $attributes = null, $placement = Container::APPEND)
+    public function __invoke(?array $attributes = null, $placement = Container::APPEND)
```

Comment:
Maybe we should add @param tags for the parameters too, since $placement doesn´t have a type assigned yet? Wdyt?

Reply:
thank you! nice catch, added the missing params